### PR TITLE
Auto-commit: Update translations file for CommCare release 2.23

### PR DESCRIPTION
**Pull request by amstone326**

### CODE REVIEW

**Comment by czue** [comment]
this is neat. is the additional trailing whitespace intentional?

**Comment by amstone326** [comment]
@czue Ah no it's not, should fix that in my script. I'll change it manually for now.

**Comment by amstone326** [comment]
Ah ok, @dannyroberts pointed out that you probably meant the extra whitespace at the end of lines. But those are just a result of of the actual file in the mobile codebase changing. The script I'm using just pulls out the entire contents of our translations files and amalgamates them into a new master translations file to replace the old one. I'm not actually using any diff-checking at all

**Comment by amstone326** [comment]
Here's the script if you're curious https://github.com/dimagi/mobile-deploy/pull/1/files

**Comment by amstone326** [comment]
@dannyroberts, @czue Is this blocking on anything to be merged?

**Comment by czue** [comment]
@amstone326 I think I was confused about whether you were going to update whitespace. sounds like it's expected though

File: messages_en-2.txt [hunk 1]
```diff
@@ -816,33 +983,28 @@ odk_change_language=Change Language
...
 odk_delete_yes=Delete Forms
 odk_discard_answer=Remove response
-odk_discard_group=Remove group
 odk_download=Get Selected
 odk_download_forms_result=Download Result
 odk_downloading_data=Connecting to Server
 odk_download_all_successful=All forms downloaded successfully.
 odk_download_complete=Form Get Complete
-odk_download_failed_with_error=Form \"${0}\" failed. ${1}.
+odk_download_failed_with_error=Form \"%1$s\" failed. %2$s.
```
Comment (czue) [inline messages_en-2.txt]: how does a change like this work with multiple versions of commcare?

**Comment by czue** [comment]
not merging due to my last question but :+1: from me assuming that changing the template structures in a bunch of these doesn't break old versions

File: messages_en-2.txt (hunk 1 above)
Comment (amstone326) [inline messages_en-2.txt]: I have no idea, going to punt to @ctsims or @wpride ?

Comment (ctsims) [inline messages_en-2.txt]: I'm confused actually, these look like they're going backwards? The commcare syntax is ${0}, not %1$

Comment (amstone326) [inline messages_en-2.txt]: @ctsims  That's not what's in our strings.xml file currently: https://github.com/dimagi/commcare-odk/blob/master/app/res/values/strings.xml#L130-L352

Comment (ctsims) [inline messages_en-2.txt]: Aliza:

Right, those are formatted in the Android style. This is what I was
referring to WRT needing to normalize the strings between the two. I think
we had done so manually with the strings on HQ currently, so it's a bit
unfortunate that we'd be regressing that behavior with the automated system.

I bet a few well-placed regex's would fix this actually, we should update
the scripts to do something like


s/\%\([\d]+)\$/\${\1}/g

to do the replacements on each line before they are pushed. I had forgotten
someone manually normalized the strings at some point.

-Clayton

**Comment by czue** [comment]
note to potential reviewers - I think this PR is blocked until @amstone326 and @ctsims reach resolution on the formatting

**Comment by amstone326** [comment]
@ctsims Should be good to go now, and the corresponding changes to the script are PR'd.

**Pull request merged**